include/crow/websocket.h

```diff
@@ -72,19 +100,40 @@ namespace crow
                     start(crow::utility::base64encode((char*)digest, 20));
 				}
 
+                /// Send data through the socket.
                 template<typename CompletionHandler>
                 void dispatch(CompletionHandler handler)
                 {
                     adaptor_.get_io_service().dispatch(handler);
                 }
 
+                /// Send data through the socket and return immediately.
                 template<typename CompletionHandler>
                 void post(CompletionHandler handler)
                 {
                     adaptor_.get_io_service().post(handler);
                 }
 
-                void send_pong(const std::string& msg)
+                /// Send a "Ping" message.
+
+                ///
+                /// Usually invoked to check if the other point is still online.
+                void send_ping(const std::string& msg) override
+                {
+                    dispatch([this, msg]{
+                        char buf[3] = "\x89\x00";
+                        buf[1] += msg.size();
```

Comment:
Maybe initialize buf with proper data - `char buf[3] {'\x89', msg.size()};`?

And are we sure that size of this message will fit into byte?

Reply:
Sure I can do that.

I did notice the one byte issue, technically no we aren't sure, but normally this isn't an issue, since sending a payload with a ping message is optional in the first place. I was planning on addressing it in a later PR.
Would you like me to address this issue here?